src/ipc-events.ts

```diff
@@ -1,3 +1,34 @@
-export enum IpcEvents {
-  CLICK_TITLEBAR_MAC = 'CLICK_TITLEBAR_MAC'
+export const enum IpcEvents {
+  ARE_YOU_BUSY = 'ARE_YOU_BUSY',
+  GET_PATH = 'GET_PATH',
+  GET_USER_AGENT = 'GET_USER_AGENT',
+  WINDOW_READY = 'WINDOW_READY',
+  CLICK_TITLEBAR_MAC = 'CLICK_TITLEBAR_MAC',
+  SHOW_OPEN_DIALOG = 'SHOW_OPEN_DIALOG',
+  SHOW_SAVE_DIALOG = 'SHOW_SAVE_DIALOG',
+  MESSAGE_BOX = 'MESSAGE_BOX',
+  CHANGE_ICON = 'CHANGE_ICON',
+  ADD_RECENT_FILE = 'ADD_RECENT_FILE',
+  FILE_DROPPED = 'FILE_DROPPED',
+  OPEN_SENTRY = 'OPEN_SENTRY',
+  PREFERENCES_SHOW = 'PREFERENCES_SHOW',
+  SET_SETTINGS = 'SET_SETTINGS',
+  FIND = 'FIND',
+  QUIT = 'QUIT'
+}
+
+export const enum TOUCHBAR_IPC {
```

Comment:
I think we can merge `TOUCHBAR_IPC` and `STATE_IPC` into the main `IpcEvents` enum

Reply:
Good idea, just did it!